### PR TITLE
GOOGLEDOCS-451

### PR DESCRIPTION
**Pull request by DenisGabriela**
- upgraded alfresco-repository and share based on acs-packaging and acs-community-packaging

### CODE REVIEW

**Comment by DenisGabriela** [comment]
> Note: Share 6.0.c is not Enterprise ? 

I don't know... Share 6.0.c is used for both enterprise and community: https://artifacts.alfresco.com/nexus/content/groups/internal/org/alfresco/acs-community-packaging/6.1.1/acs-community-packaging-6.1.1.pom
and
https://artifacts.alfresco.com/nexus/content/groups/internal/org/alfresco/acs-packaging/6.1.0-EA3/acs-packaging-6.1.0-EA3.pom

**Comment by montgolfiere** [comment]
Actually, I think you're right - there is a single version of Share. Let's use 6.1.0-EA3 now and watch master branch (for 6.1.0-RC1 etc). We should try to remember to come back and update again, if possible.

**Comment by DenisGabriela** [comment]
I've just updated alfresco-repository with the versions used on master (on acs-packaging and acs-community-packaging).

File: pom.xml [hunk 1]
```diff
@@ -20,8 +20,8 @@
       <alfresco.max.version>6.99.99</alfresco.max.version>
       <alfresco.google.config.version>1.0.0</alfresco.google.config.version>
       <alfresco.google.config.classifier>-community</alfresco.google.config.classifier>
-      <compiler-plugin.version>3.1</compiler-plugin.version>
-      <java.version>1.8</java.version>
+      <compiler-plugin.version>3.8.0</compiler-plugin.version>
+      <java.version>11</java.version>
```
Comment (montgolfiere) [inline pom.xml]: This should remain as 1.8 (to ensure 1.8 compatible byte code).

Comment (DenisGabriela) [reply pom.xml]: Ok, I will revert it.

Comment (DenisGabriela) [reply pom.xml]: Done. Thanks.

**Pull request merged**